shipyard_blueprints/postgresql/shipyard_postgresql/cli/execute_sql.py

```diff
@@ -14,21 +19,9 @@ def get_args():
     parser.add_argument("--database", dest="database", required=False)
     parser.add_argument("--port", dest="port", default="5432", required=False)
     parser.add_argument("--url-parameters", dest="url_parameters", required=False)
-    parser.add_argument("--db-connection-url", dest="db_connection_url", required=False)
```

Comment:
I am assuming this is not an input in the current blueprint,correct?

Reply:
correct